src/ArrayToXml.php

```diff
@@ -99,6 +107,22 @@ public function toDom()
         return $this->document;
     }
 
+    /**
+     * Sets dom properties on $this->document.
+     *
+     * @param $domProperties
+     * @throws \Exception
+     */
+    public function setDomProperties($domProperties)
+    {
+        foreach ($domProperties as $key => $value) {
+            if (! property_exists($this->document, $key)) {
+                throw new \Exception($key.' is not a valid property of DOMDocument', 1561114534);
```

Comment:
What is `1561114534`? Do we really need this?

Reply:
@freekmurze i tend to give every custom exception a "unique" error code (i use the current timestamp for that). Makes it easier to apply certain reactions when catching it. But it is not a 100% neccesary just in case. You want me to remove that?.

Reply:
Yes, just remove it.

Reply:
Also import the Exception namespace so we can get rid of that preceding `\`.

Reply:
@freekmurze done